src/sentry/search/snuba/executors.py

```diff
@@ -1877,5 +1892,10 @@ def query(
 
         groups = group_queryset.in_bulk(paginator_results.results)
         paginator_results.results = [groups[k] for k in paginator_results.results if k in groups]
+        metrics.timing(
```

Comment:
why not just use @metrics.wraps on the whole function?

Reply:
smart!